### PR TITLE
Blog post on Sheffield's first meeting

### PR DESCRIPTION
**Pull request by hertzsprung**

### CODE REVIEW

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "Sheffield ReproducibiliTea: what happened in our first meeting?"
+date:   2019-11-18 17:00:00 +0100
+author: James Shaw
+categories: [Journal club]
+tags: [Sheffield]
+---
+
+Sheffield University held their first ReproducibiliTea meeting in October 2019.  We had about 15 people come along from medicine, biology, engineering and psychology, with [Stavrina Dimosthenous](https://twitter.com/stavrhina), a PhD researcher in materials science, leading the discussion.  Stavrina asked us: [Is the Replicability Crisis Overblown? Three Arguments Examined](https://doi.org/10.1177/1745691612463401), a psychology paper by Pashler and Harris, 2012.  We discussed what [p-values](https://en.wikipedia.org/wiki/P-value) are, and how [p-hacking](https://en.wikipedia.org/wiki/Data_dredging) can arise when researchers discard “outlier” data that doesn’t support their hypothesis.
```
Comment (katiedrax) [inline _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: Would you mind introducing yourself so the reader knows who the 'we' is? It also gives you an opportunity to give yourself a bit of promo for your awesome work!

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Hypothesis testing led us to think about what happens when a researcher finds a [negative result](https://en.wikipedia.org/wiki/Null_result), when the data fails to confirm the hypothesis.  In our meeting, some people had seen negative outcomes of clinical trials published in “[high impact](https://en.wikipedia.org/wiki/Impact_factor)” journals, because these negative outcomes risked harming patients, but not when results show no clear outcome either way.  Some attendees felt that experimental design and publication quality was generally high in their area of research.  But in other research areas, some attendees thought quality was more variable, and questioned whether journals and [peer reviewers](https://en.wikipedia.org/wiki/Peer_review) had sufficient incentive (or time) to ensure published research is high quality.
```
Comment (katiedrax) [inline _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: Change to "Our discussion about hypothesis testing"?
Can you be specific about which research areas you're talking about?

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+The next [Sheffield ReproducibiliTea](/journal-clubs/#Sheffield) is on November 14, see you there!
```
Comment (katiedrax) [inline _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: Time, paper, room? Or maybe contact details so people know how to find out this info?

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+Hypothesis testing led us to think about what happens when a researcher finds a [negative result](https://en.wikipedia.org/wiki/Null_result), when the data fails to confirm the hypothesis.  In our meeting, some people had seen negative outcomes of clinical trials published in “[high impact](https://en.wikipedia.org/wiki/Impact_factor)” journals, because these negative outcomes risked harming patients.  But [results don't tend to get published](https://en.wikipedia.org/wiki/Publication_bias) when they show no clear outcome either way.  Some attendees felt that experimental design and publication quality was generally high in their area of research.  In other research areas, some attendees thought quality was more variable, and questioned whether journals and [peer reviewers](https://en.wikipedia.org/wiki/Peer_review) had sufficient incentive (or time) to ensure published research is high quality.
```
Comment (katiedrax) [inline _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: Change to "Our discussion about hypothesis testing"?
Can you be specific about which research areas you're talking about?

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md (hunk 1 above)
Comment (mjaquiery) [reply _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: FYI: if you want to link to your JC (e.g. when mentioning its name) you can do so using the path relative to the site root:

```suggestion
[Sheffield University](/journal-clubs/#Sheffield)
```

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md (hunk 3 above)
Comment (hertzsprung) [reply _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: Now reads:

> The next [Sheffield ReproducibiliTea](/journal-clubs/#Sheffield) is at 1pm 14th November, Pam Liversidge E06, where we'll be covering a recent opinion piece by Daniele Fanelli, 2018: [Is science really facing a reproducibility crisis, and do we need it to?](https://doi.org/10.1073/pnas.1708272114)  Log into your Sheffield University account, and you can [join our Google group](https://bit.ly/tuosrepro) and [subscribe to our Google calendar](https://calendar.google.com/calendar?cid=c2hlZmZpZWxkLmFjLnVrX3BxYnVmMWQzNDk5dXVyaTgzZDJyYjVvYXJzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).  See you there!

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md (hunk 4 above)
Comment (hertzsprung) [reply _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: Done: clinical researchers thought experiments and publication quality was generally good -- which is reassuring!

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md (hunk 1 above)
Comment (hertzsprung) [reply _posts/2019-11-18-Sheffield-ReproducibiliTea.md]: > Would you mind introducing yourself so the reader knows who the 'we' is? It also gives you an opportunity to give yourself a bit of promo for your awesome work!

Added a quick bio at the bottom of the post and reworded the opening sentences.  I would rather focus on the event than myself :relaxed:

File: _posts/2019-11-18-Sheffield-ReproducibiliTea.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+title:  "Sheffield ReproducibiliTea: what happened in our first meeting?"
+date:   2019-10-18 16:00:00 +0100
+author: James Shaw
+categories: [Journal club]
+tags: [Sheffield]
+---
+
+[Sheffield University](/journal-clubs/#Sheffield) held its first ReproducibiliTea meeting in October 2019, with about 15 people attending from medicine, biology, engineering,  psychology, and other departments.
+[Stavrina Dimosthenous](https://twitter.com/stavrhina), a PhD researcher in materials science, lead the discussion and introduced us to a psychology paper by Pashler and Harris, 2012: [Is the Replicability Crisis Overblown? Three Arguments Examined](https://doi.org/10.1177/1745691612463401).
+We discussed what [p-values](https://en.wikipedia.org/wiki/P-value) are, what [p-hacking](https://en.wikipedia.org/wiki/Data_dredging) is, and how p-hacking can arise when researchers discard “outlier” data that doesn’t support their hypothesis.
```
Comment (katiedrax) [inline _posts/2019-11-18-Sheffield-ReproducibiliTea.md]:
```suggestion
We discussed what [p-values](https://en.wikipedia.org/wiki/P-value) are, what [p-hacking](https://en.wikipedia.org/wiki/Data_dredging) is, and how p-hacking can arise when researchers exclude data that doesn’t support their hypothesis.
```

**Pull request merged**